concordium-node/src/utils.rs

```diff
@@ -10,7 +10,11 @@ use std::{
 };
 
 pub fn to_hex_string(bytes: &[u8]) -> String {
```

Comment:
Just remove this function. It is not used anywhere. It's some leftover from times where we did not use types from base for hashes.